consensus-types/blocks/execution.go

```diff
@@ -647,10 +647,6 @@ func PayloadToHeaderCapella(payload interfaces.ExecutionData) (*enginev1.Executi
 // IsEmptyExecutionData checks if an execution data is empty underneath. If a single field has
 // a non-zero value, this function will return false.
 func IsEmptyExecutionData(data interfaces.ExecutionData) (bool, error) {
-	_, ok := data.Proto().(*enginev1.ExecutionPayloadCapella)
-	if ok {
-		return false, nil
-	}
```

Comment:
This was a bad check. `data` is an interface, and the underlying object could be bellatrix or capella